packages/yourloops/pages/patient/teams/add-dialog.tsx

```diff
@@ -25,31 +25,6 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import _ from 'lodash'
-import React from 'react'
-import { Trans, useTranslation } from 'react-i18next'
-import { type Theme } from '@mui/material/styles'
-
-import { makeStyles } from 'tss-react/mui'
-
-import Box from '@mui/material/Box'
-import Button from '@mui/material/Button'
-import Checkbox from '@mui/material/Checkbox'
-import CircularProgress from '@mui/material/CircularProgress'
-import Dialog from '@mui/material/Dialog'
-import DialogActions from '@mui/material/DialogActions'
-import DialogContent from '@mui/material/DialogContent'
-import DialogContentText from '@mui/material/DialogContentText'
-import DialogTitle from '@mui/material/DialogTitle'
-import FormControl from '@mui/material/FormControl'
-import FormControlLabel from '@mui/material/FormControlLabel'
-import InputLabel from '@mui/material/InputLabel'
-import Link from '@mui/material/Link'
-import TextField from '@mui/material/TextField'
-import metrics from '../../../lib/metrics'
-import { getDisplayTeamCode, REGEX_TEAM_CODE_DISPLAY, type Team, useTeam } from '../../../lib/team'
-import { diabeloopExternalUrls } from '../../../lib/diabeloop-urls.model'
-
 interface AddTeamDialogContentProps {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏This can also be removed (this is used in a test file but is useless)